modules/images/regenerate-existing-images/load.php

```diff
@@ -7,3 +7,74 @@
  * @since n.e.x.t
  * @package performance-lab
  */
+
+/**
+ * Registers the background job taxonomy.
+ *
+ * Intentionally on lower priority, so that other post types can be
+ * registered successfully and taxonomy can be used for all the post types.
+ *
+ * This taxonomy will be used as a queue system where each term represents a job in the queue.
+ *
+ * @since n.e.x.t
+ */
+function perflab_register_background_job_taxonomy() {
+	// Labels for the background job taxonomy.
+	$labels = array(
+		'name'                  => _x( 'Background Jobs', 'taxonomy general name', 'performance-lab' ),
+		'singular_name'         => _x( 'Background Job', 'taxonomy singular name', 'performance-lab' ),
+		'search_items'          => __( 'Search Background Jobs', 'performance-lab' ),
+		'all_items'             => __( 'All Background Jobs', 'performance-lab' ),
+		'parent_item'           => __( 'Parent Background Job', 'performance-lab' ),
+		'parent_item_colon'     => __( 'Parent Background Job:', 'performance-lab' ),
+		'edit_item'             => __( 'Edit Background Job', 'performance-lab' ),
+		'update_item'           => __( 'Update Background Job', 'performance-lab' ),
+		'add_new_item'          => __( 'Add New Background Job', 'performance-lab' ),
+		'new_item_name'         => __( 'New Background Job Name', 'performance-lab' ),
+		'menu_name'             => __( 'Background Jobs', 'performance-lab' ),
+		'view_item'             => __( 'View Background Job', 'performance-lab' ),
+		'not_found'             => __( 'Background Job Not Found', 'performance-lab' ),
+		'no_terms'              => __( ' No Background Jobs', 'performance-lab' ),
+		'items_list'            => __( 'Background Jobs List', 'performance-lab' ),
+		'items_list_navigation' => __( 'Background Jobs List Navigation', 'performance-lab' ),
+		'back_to_items'         => __( 'Back to Background Jobs', 'performance-lab' ),
+	);
+
+	// Taxonomy arguments.
+	$args = array(
+		'labels'       => $labels,
+		'public'       => false,
+		'show_ui'      => true,
+		'show_in_rest' => false,
```

Comment:
Nit-pick, technically no need to pass `show_in_rest` since it's `false` anyway by default. But also perfectly fine to keep it.